codegen/module.go

```diff
@@ -1267,7 +1267,11 @@ func (system *ModuleSystem) IncrementalBuild(
 	for _, moduleList := range toBeBuiltModules {
 		moduleCount += len(moduleList)
 	}
-
+	qpsLevels, err := PopulateQPSLevels(baseDirectory + "/endpoints")
+	if err != nil {
+		fmt.Printf("Error in populating qps levels")
```

Comment:
I think you can do something like this here: 

`return nil, return nil, errors.Wrapf(err, "error in populating qps levels")